### ISSUE TITLE
Fails to install with vagrant 1.9.6

### ISSUE THREAD

**Issue by greenaj** [Parallels/vagrant-parallels#301]
On OS X Sierra even if I install libxml2 with homebrew I still get:

Building nokogiri using system libraries.
pkg-config could not be used to find libxml-2.0
Please install either `pkg-config` or the pkg-config gem per

    gem install pkg-config -v "~> 1.1.7"

pkg-config could not be used to find libxslt
Please install either `pkg-config` or the pkg-config gem per

    gem install pkg-config -v "~> 1.1.7"

pkg-config could not be used to find libexslt
Please install either `pkg-config` or the pkg-config gem per

    gem install pkg-config -v "~> 1.1.7"

ERROR: cannot discover where libxml2 is located on your system. please make sure `pkg-config` is installed.
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

**Comment by legal90** [comment]
Confirmed. Unfortunately, our plugin could not be installed on Vagrant 1.9.6 at all because of the bug in Vagrant: https://github.com/mitchellh/vagrant/issues/8738

The fix is already merged: https://github.com/mitchellh/vagrant-installers/pull/114 , so look forward to the next Vagrant release.

At the moment I can recommend rolling back to Vagrant 1.9.5 ([download link](https://releases.hashicorp.com/vagrant/1.9.5/vagrant_1.9.5_x86_64.dmg)) and then installing our plugin with `NOKOGIRI_USE_SYSTEM_LIBRARIES` set:

```
$ NOKOGIRI_USE_SYSTEM_LIBRARIES=true vagrant plugin install vagrant-parallels
```

**Comment by legal90** [comment]
The problem was solved in Vagrant 1.9.7 release.
